Handlers/ShowroomHandlers.js

```diff
@@ -283,8 +290,10 @@ function getStatusForEvents (allEvents, mainCallback) {
                                 let attendeeUserIDs = [];
                                 //Only add user if they are participating in some way
                                 attendeeObjs.forEach((obj) => {
-                                    if ((obj.isListeningOnly && obj.isListeningOnly == true )|| (obj.hasJoinedVoice && obj.hasJoinedVoice == true) || (obj.hasVideo && obj.hasVideo == true)){
-                                        attendeeUserIDs.push(obj.userID);
+                                    if (obj != undefined) { //Let's prevent "can't read from undefined" errors
+                                        if ((obj.isListeningOnly && obj.isListeningOnly == true )|| (obj.hasJoinedVoice && obj.hasJoinedVoice == true) || (obj.hasVideo && obj.hasVideo == true)){
```

Comment:
For the boolean operation checking that var exists is enough to also verify that it is true.

Leave as is if works and for assignee’s sanity

Reply:
haha, yes, just tested again, funcionea